src/quicknxs/interfaces/data_handling/instrument.py

```diff
@@ -249,13 +249,20 @@ def load_data(self, file_path, configuration=None):
             missing_keys = (polarizer > 0 and self.pol_state not in event_ws.getRun()) or (
                 analyzer > 0 and self.ana_state not in event_ws.getRun()
             )
+            # If running in unpolarized mode, no filtering is needed
+            unpolarized = polarizer == 0 and analyzer == 0
```

Comment:
```suggestion
            unpolarized = (polarizer == 0 and analyzer == 0)
```

Reply:
I wrote if with parentheses initially, but ruff didn't allow it :disappointed:

Reply:
stupid ruff :exclamation: